core/src/components/checkbox/checkbox.scss

```diff
@@ -260,6 +309,8 @@ input {
  */
 :host(.checkbox-label-placement-stacked) .checkbox-wrapper {
   flex-direction: column;
+
+  text-align: center;
```

Comment:
These text alignment changes are necessary because without it the label is left aligned and when you change between helper/error text it jumps:


https://github.com/user-attachments/assets/c396a00a-d25c-4970-8e31-1176d94ea22c

After my changes:

https://github.com/user-attachments/assets/16fa1e52-2d73-4093-99f4-05faa8961810

This issue does not exist without helper/error text.

Reply:
This makes sense to me. Everything is centered already so the supporting text should follow that.